homeassistant/components/tuya/sensor.py

```diff
@@ -734,6 +734,54 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             native_unit_of_measurement=UnitOfElectricPotential.VOLT,
             subkey="voltage",
         ),
+        TuyaSensorEntityDescription(
+            key=DPCode.VO,
+            name="Voltage",
+            device_class=SensorDeviceClass.VOLTAGE,
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+            subkey="voltage",
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.CTIME,
+            name="Total charging time",
+            icon="mdi:history",
+            state_class=SensorStateClass.TOTAL_INCREASING,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.CO,
+            name="Temperature",
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.KW,
+            name="Power",
+            device_class=SensorDeviceClass.POWER,
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement=UnitOfPower.KILO_WATT,
+            subkey="power",
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.CPOWER,
+            name="Total energy",
+            device_class=SensorDeviceClass.POWER,
+            state_class=SensorStateClass.TOTAL_INCREASING,
```

Comment:
Something is wrong here.  Power is not energy, also power should not be `TOTAL_INCREASING`

Reply:
Energy is kWh

kW/h is something completely different.

Reply:
Typo, i wanted to say "kwh"